Rakefile

```diff
@@ -42,4 +42,4 @@ end
 
 Kitchen::RakeTasks.new
 
-task default: [:spec, :make_bin_executable, :yard, :rubocop, :check_binstubs]
```

Comment:
we should keep default and add a `quick` like this: https://github.com/sensu-plugins/sensu-plugins-load-checks/blob/4.0.0/Rakefile#L46 Basically it is default minus intergration which I see is also missing.